testing/integration/upgrade_fleet_test.go

```diff
@@ -426,3 +573,89 @@ func agentUpgradeDetailsString(a *kibana.AgentExisting) string {
 
 	return fmt.Sprintf("%#v", *a.UpgradeDetails)
 }
+
+// prepareFileServer prepares and returns a started HTTPS file server serving
+// files from rootDir and using cert as its TLS certificate.
+func prepareFileServer(t *testing.T, rootDir string, cert tls.Certificate) *httptest.Server {
```

Comment:
it's better if we say what we actually do in this function
```suggestion
func startHttpsFileServer(t *testing.T, rootDir string, cert tls.Certificate) *httptest.Server {
```